api/v1/storagecluster_types.go

```diff
@@ -393,6 +394,18 @@ type EncryptionSpec struct {
 	KeyManagementService KeyManagementServiceSpec `json:"kms,omitempty"`
 }
 
+type SecuritySpec struct {
+	// +optional
+	KMS KMSSecuritySpec `json:"kms,omitempty"`
+}
+
+type KMSSecuritySpec struct {
+	// +optional
+	EnableKeyRotation bool `json:"enableKeyRotation,omitempty"`
+	// +optional
+	Schedule string `json:"schedule,omitempty"`
+}
```

Comment:
We already have `KeyManagementService` just above this. Can't these spec be added there?